File: 119.Pascal's Triangle2.py
class Solution(object):
    def getRow(self, rowIndex):
        """
        :type rowIndex: int
        :rtype: List[int]
        """
        n = rowIndex + 1
        res = [1] * (n)
        for i in range(2, n):
            for j in range(i-1, 0, -1):
                res[j] = res[j] + res[j-1]
        return res

File: 119.Pascal's Triangle2.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        rowIndex++;
        vector<int> res (rowIndex,1);
        for(int i=2; i<rowIndex; i++){
            for(int j=i-1; j>=1; j--){
                res[j] += res[j-1];
            }
        }
        return res;
    }
};
